doc/project/release_process.rst

```diff
@@ -85,24 +85,155 @@ At that point, the whole process starts over again.
++----------+------------------------------------------------------------------+
+| Major    | As with critical, but an acceptable alternative exists.          |
++----------+------------------------------------------------------------------+
+| Moderate | Causes the system to produce incorrect, incomplete, or           |
+|          | inconsistent results.                                            |
++----------+------------------------------------------------------------------+
+| Minor    | No crashes or damage to usability of the system and desired      |
+|          | results can be easily obtained by working around the defects.    |
++----------+------------------------------------------------------------------+
+| Cosmetic | Related to look and feel of the system.                          |
++----------+------------------------------------------------------------------+
+
+Severity shall be set based on the context of the bug and not in relation to the
+overall impact on the codebase. For example, a bug in an optional subsystem that
+results in a crash shall be marked as `critical` even if the bug does not impact
+every user of Zephyr.
+
+
+Blocker Bugs
+============
+
+Blocker bug process kicks during the release process and is in effect after the
+feature freeze milestone. An issue labeled as a blocker practically blocks a
+release from happening. All blocker bugs shall be resolved before a release is
+created.
 
-The current backlog of prioritized bugs shall be used as a quality metric to
-gate the final release. The following counts shall be used:
+After hard freeze (RC3), a change to the codebase can only be merged to ‘main'
+and included in the release if it fixes a bug that is granted blocker or freeze
+exception status.
 
-.. csv-table:: Bug Count Release Thresholds
-   :header: "High", "Medium", "Low"
-   :widths: auto
+Bugs of moderate severity and higher are typically the candidates to be promoted
+to blocker bugs.
 
+The following guidelines shall be followed for release blocker bugs:
 
-   "0","<20","<50"
+- Only mark bugs as blockers if the software (Zephyr) must not be released with the bug present.
+- All collaborators can add or remove blocking labels.
+- Evaluate bugs as potential blockers based on their severity and prevalence.
+- Provide detailed rationale whenever adding or removing a blocking label.
+- Ensure all blockers have the milestone tagged.
+- Release owners have final say on blocking status; contact them with any questions.
 
-.. note::
+Hardware Support Tiers
+======================
+
+Tier 0: Emulation Platforms
+---------------------------
+
+- Failures can block Pull Requests.
+- Supported by the Zephyr project itself, commitment to fix bugs in releases.
+- One is required for each new architecture.
+- Bugs reported against platforms of this tier are to be evaluated and treaeted as
+  a general bug in Zephyr and should be dealt with the highest priority.
```

Comment:
Highest priority should they have the "blocker" modifier?

Reply:
they could be a blocker if there is a justification, I would not say they are automatically blockers.

Reply:
> but this is where I get mentally caught up with "priority" "severity" "impact" "blocker". Too many terms with too many implied meanings

not sure why you are putting impact in here as if it is something else we have to track, this is just the same as severity, I mentioned that in the context of coverity, but Impact=Severity. 

Blocker is used to mark any bug or issue of any severity as something that needs to be addressed before a release, everything else can be addressed later and should not block the release. 

Some critical bug in an optional subsystem might not be considered as something that should block a release, if it should, it will get the blocker label, but not all critical bugs are blockers.

Reply:
These lines on the Tier criteria setting a priority also confuse me.  Especially since line 134 of this document states:

> Bug priorities can be used by owners of bugs in an area or subsystem to manage
their workflow and priorities.

But here (and in the lower Tier definitions) the document is specifically stating a priority for the bug/issue filed.  Is s/priority/severity needed here?

Reply:
We should be able to use the term priority without having a label for it, to avoid any confusion maybe use urgency?